go-controller/hack/test-go.sh

```diff
@@ -61,7 +61,7 @@ function testrun {
     fi
     if grep -q "ginkgo" ."${path}"/*_test.go; then
 	    prefix=$(echo "${path}" | cut -c 2- | sed 's,/,_,g')
-        ginkgoargs="-ginkgo.v ${ginkgo_focus} -ginkgo.reportFile ${TEST_REPORT_DIR}/junit-${prefix}.xml"
+        ginkgoargs="-ginkgo.v ${ginkgo_focus} --ginkgo.junit-report ${TEST_REPORT_DIR}/junit-${prefix}.xml"
```

Comment:
I would just use single dash here as with the other ginkgo parameter

Reply:
done